cpp/src/arrow/acero/options.h

```diff
@@ -106,7 +106,7 @@ class ARROW_ACERO_EXPORT SourceNodeOptions : public ExecNodeOptions {
 ///
 /// The table source node will slice off chunks, defined by `max_batch_size`
 /// for parallel processing.  The source node extends source node and so these
```

Comment:
I wasn't sure, but should this be changed too?

```suggestion
/// for parallel processing.  The table source node extends source node and so these
```

Reply:
I think we should

Reply:
Change accepted, thanks.